src/request.rs

```diff
@@ -445,6 +451,24 @@ impl Request for Rename {
     const METHOD: &'static str = "textDocument/rename";
 }
 
+#[derive(Debug)]
+pub enum DocumentColor {}
+
+impl Request for DocumentColor {
+    type Params = DocumentColorParams;
+    type Result = Option<Vec<ColorInformation>>;
+    const METHOD: &'static str = "textDocument/documentColor";
+}
+
+#[derive(Debug)]
+pub enum ColorPresentation {}
+
+impl Request for ColorPresentation {
+    type Params = ColorPresentationParams;
+    type Result = Option<Vec<ColorPresentation>>;
```

Comment:
Ditto

Reply:
Travis fails because you are reffering to `pub enum ColorPresentation {}` here :). I think it is fine to have the same name for them, just refer to the deserializable struct explicitly